prov/efa/src/rxr/rxr_ep.c

```diff
@@ -104,6 +104,7 @@ struct rxr_rx_entry *rxr_ep_alloc_rx_entry(struct rxr_ep *ep, fi_addr_t addr, ui
 	rx_entry->window = 0;
 	rx_entry->unexp_pkt = NULL;
 	rx_entry->rma_iov_count = 0;
+	memset(rx_entry->desc, 0, sizeof(rx_entry->desc));
```

Comment:
Why is memsetting here, but initializing everything above to 0/NULL the right choice?  That seems terribly inefficient.

Reply:
I was using a for loop to initialize each rx_entry->desc[i] to NULL before, and changed to memset base on team's feedback, mainly because it's conciser.

Reply:
I think if we memset the whole `rx_entry` to 0 at the beginning, then we do not need to set invidiual field to 0.

Reply:
I was misunderstanding then. I will update it to memset the whole rx_entry to 0 at the beginning.